src/messaging/tests/MessagingContext.h

```diff
@@ -94,10 +96,11 @@ class MessagingContext
     secure_channel::MessageCounterManager mMessageCounterManager;
     IOContext * mIOContext;
 
-    NodeId mBobNodeId    = 123654;
-    NodeId mAliceNodeId  = 111222333;
-    uint16_t mBobKeyId   = 1;
-    uint16_t mAliceKeyId = 2;
+    NodeId mBobNodeId        = 123654;
+    NodeId mAliceNodeId      = 111222333;
+    uint16_t mBobKeyId       = 1;
+    uint16_t mAliceKeyId     = 2;
+    uint16_t mFriendsGroupId = 3;
```

Comment:
Shouldn't this be a GroupId?  And I'd set it to maybe 517 or some such random value that is not going to accidentally overlap with other things....

Reply:
Done

Reply:
fixed